js/contextMenus.js

```diff
@@ -543,17 +543,19 @@ function getMisspelledSuggestions (selection, isMisspelled, suggestions) {
     if (isMisspelled) {
       template.push({
         label: locale.translation('learnSpelling'),
-        click: () => {
-          appActions.addWord(selection, true)
-          // This is needed so the underline goes away
-          webviewActions.replace(selection)
+        click: (item, focusedWindow) => {
+          if (focusedWindow) {
+            focusedWindow.webContents.addWord(selection)
```

Comment:
`focusedWindow` should be deprecated and context menu items should use actions by default. I would use something like `appActions.learnSpelling` and `appActions.forgetLearnedSpelling` and pass the tabId (always set in muon `HandleContextMenu` and retrieved from nodeProps.properties)